.travis.yml

```diff
@@ -22,7 +22,7 @@ install:
   - source activate test-env
   - if [[ $DEP_VERSIONS == "oldest" ]]; then
       if [[ $TRAVIS_PYTHON_VERSION == 2.7 ]]; then
-        conda install numpy==1.7.0 scipy==0.11.0 sympy==0.7.4.1 cython==0.17 theano==0.6.0;
+        conda install numpy==1.7.0 scipy==0.11.0 sympy==0.7.4.1 cython==0.17 theano==0.7.0;
```

Comment:
I just added a NumPy 1.7 compatible Theano for Linux 64 here:

https://anaconda.org/pydy/theano

You can change this to `conda install -c pydy numpy==1.7.0 scipy==0.11.0 sympy==0.7.4.1 cython==0.17 theano==0.7.0;` to pull packages from the main anaconda repos and our pydy channel. I think that will fix this.